kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/config/KafkaSchedulerConfiguration.java

```diff
@@ -181,15 +188,26 @@ public String toString() {
         @Override
         public KafkaSchedulerConfiguration parse(byte[] bytes) throws ConfigStoreException {
             try {
-                return JsonUtils.MAPPER.readValue(bytes, KafkaSchedulerConfiguration.class);
+
+                final ObjectMapper mapper = Jackson.newObjectMapper()
+                        .registerModule(new GuavaModule())
+                        .registerModule(new JavaTimeModule())
+                        .registerModule(new Jdk8Module());
+                return  mapper.readValue(bytes, KafkaSchedulerConfiguration.class);
```

Comment:
should be able to use `SerializationUtils.fromJsonString(bytes, KafkaSchedulerConfiguration.class)`